src/init-action.ts

```diff
@@ -546,6 +546,20 @@ async function run() {
       }
     }
 
+    // Set CODEQL_EXTRACTOR_CPP_BUILD_MODE_NONE
+    if (config.languages.includes(Language.cpp)) {
+      const bmn_var = "CODEQL_EXTRACTOR_CPP_BUILD_MODE_NONE";
```

Comment:
This is probably a linter error. Here's a more canonical name.

```suggestion
      const bmnVar = "CODEQL_EXTRACTOR_CPP_BUILD_MODE_NONE";
```

Reply:
Is this simpler? `features.getValue` is guaranteed to return a boolean and if the env var is already set, it will use that as an override.

```js
const bmnVar = await features.getValue(Feature.CppBuildModeNone, codeql);
core.exportVariable(bmnVar);
logger.info(`Setting C++ build-mode: none to ${bmnVar}`);
```